### PR TITLE
Add an apply button for updating prefs and restarting the k8s engine

### PR DESCRIPTION
**Pull request by ericpromislow**
* The apply button should be enabled only when changes have been made.

There's a new TODO: we should add a status bar in the UI to indicate
when prefs have been updated or k8s has been restarted.

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ module.exports = {
     '@vue/cli-plugin-babel/preset'
   ],
   plugins: ["@babel/plugin-proposal-private-methods",
+    ["babel-plugin-root-import", { "rootPathSuffix": "./", "rootPathPrefix": "@/"}],
```
Comment (mook-as) [inline babel.config.js:6]: Do we need this back in, now that we have `module-resolver`?

File: package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "sass": "^1.32.2",
     "semver": "^7.3.4",
     "sudo-prompt": "^9.2.1",
+    "t": "^0.5.1",
```
Comment (mook-as) [inline package.json:31]: We don't seem to be using this in this patch?

**Review by mook-as — CHANGES_REQUESTED** [review]
Do we have a new issue filed to handle the status bar or whatever?

File: babel.config.js (hunk 1 above)
Comment (ericpromislow) [reply babel.config.js:6]: Status bar issue: https://github.com/mattfarina/rd/issues/90

And the `babel-plugin-root-import` field got dropped in a different branch.

File: package.json (hunk 2 above)
Comment (ericpromislow) [reply package.json:31]: I don't know how that got in there, and it didn't survive the latest rebase against `main`. But if we want to pull in modules from `dashboard` without changing them, we'll need to implement it anyway.

**Comment by ericpromislow** [comment]
This change is obsoleted by PR 98

**Pull request closed without merging**